modules/fakturabank/model/fakturabank.class.php

```diff
@@ -1443,6 +1441,8 @@ public function registerincoming() {
                         $datalineH['Vat']               = $line->TaxTotal->TaxSubtotal[0]->Percent;
                         #$datalineH['VatID']             = $line->Price->PriceAmount; #Denne mŒ nok mappes
 
+                        $datalineH['UnitCostPrice'] = $datalineH['UnitCustPrice'] + $datalineH['TaxAmount']/$Quantity;
```

Comment:
TaxAmount is for Total for invoice line.
Add a comment to clarify this.
Check if Quantity can be 0.